spec/Classes.tex

```diff
@@ -521,20 +536,11 @@ \section{Inheritance}
 \index{derived class}
 \index{classes!derived}
 
-A \emph{derived} class can inherit from one or more other classes by
-listing those classes in the derived class declaration.
-
-%When inheriting from multiple base classes, only one of the base classes
-%may contain fields.  The other classes can only define methods.  Note
-%that a class can still be derived from a class that contains fields
-%which is itself derived from a class that contains fields.
-
-%REVIEW: vass: the below ("tree") does not match the above ("multiple base classes")
-
-%These restrictions on inheritance induce a class hierarchy which has the form of
-%a tree.  A variable referring to an instance of class \chpl{C} can be
-%cast to any type that is an ancestor of \chpl{C}.  Note that casts to more- and
-%less-derived classes are both permitted.
+A class inherits, or \emph{derives}, from the class specified in the class
```

Comment:
Do other parts of the spec use the term "derives" ? It's not my favorite term and I find it confusing. Could we just stop using it here (and use inherits from instead)?

Reply:
Good suggestion. Leaving it as future work because the spec has ~25 uses of "derived" or similar.